package.json

```diff
@@ -11,7 +11,8 @@
         "discord.js": "discordjs/discord.js#stable",
         "dotenv-safe": "^8.2.0",
         "humanize-duration": "^3.21.0",
-        "reflect-metadata": "^0.1.13"
+        "reflect-metadata": "^0.1.13",
+        "rm": "^0.1.8"
```

Comment:
How is an [ORM](https://www.npmjs.com/package/rm) related to cookiecord? Probably a mistake - please revert

Reply:
Sorry Ron this is my fault, Boon had a problem which needed rm to fix but forgot to delete after.